### PR TITLE
Ws 017

### PR DESCRIPTION
**Pull request by Annie1790**
Hello @AndrewYHuang ,

Can you please have a look on my changes?
**Desktop:**
Screenshot from 2023-10-10 16-05-51

**Mobile:**
Screenshot from 2023-10-10 16-06-19

### CODE REVIEW

File: src/main/resources/static/css/login.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+@media (min-width: 640px) {
+        .label-container {font-size: 24px;}
+        .input-field {min-width: 220px;}
+        .form-container {gap: 1.25rem;}
+        .invalid-state {font-size: 14px;}
+        .warning-state {font-size: 14px;}
+        .login-button {font-size: 20px;}
+	}
```
Comment (TimLeach635) [inline src/main/resources/static/css/login.css]: Consistent indenting, and prefer spaces within curly braces
```suggestion
    .label-container { font-size: 24px; }
    .input-field { min-width: 220px; }
    .form-container { gap: 1.25rem; }
    .invalid-state { font-size: 14px; }
    .warning-state { font-size: 14px; }
    .login-button { font-size: 20px; }
}
```

File: src/main/resources/static/css/login.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+@media (max-width: 640px) {
+        .label-container {font-size: 14px;}
+        .input-field {min-width: 150px;}
+        .form-container {gap: 0.75rem;}
+        .invalid-state {font-size: 10px;}
+        .warning-state {font-size: 10px;}
+        .login-button {font-size: 14px;}
+    }
```
Comment (TimLeach635) [inline src/main/resources/static/css/login.css]: As above
```suggestion
    .label-container { font-size: 14px; }
    .input-field { min-width: 150px; }
    .form-container { gap: 0.75rem; }
    .invalid-state { font-size: 10px; }
    .warning-state { font-size: 10px; }
    .login-button { font-size: 14px; }
}
```

File: src/main/resources/static/css/login.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+.login-button > input {
+    font-weight: 400;
+	text-align: center;
+	border: 1px solid transparent;
+	padding: .375rem .75rem;
+	line-height: 1.5;
+	border-radius: .25rem;
+	color: #fff;
+	background-color: #007bff;
```
Comment (TimLeach635) [inline src/main/resources/static/css/login.css]: Consistent indenting
```suggestion
	  text-align: center;
	  border: 1px solid transparent;
	  padding: .375rem .75rem;
	  line-height: 1.5;
	  border-radius: .25rem;
	  color: #fff;
	  background-color: #007bff;
```

File: src/main/resources/static/css/login.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+.login-button {
+    margin-left: auto;
+}
+
+.login-button > input {
+    font-weight: 400;
+	text-align: center;
+	border: 1px solid transparent;
```
Comment (TimLeach635) [inline src/main/resources/static/css/login.css]: A transparent border sort of does the same thing as extra padding. Why do you have a transparent border and not just add a bit more padding?

File: src/main/resources/static/css/login.css [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    z-index: -1;
+}
+
+.invalid-state {
+    color: crimson;
+}
+
+.warning-state {
+    color: grey;
+}
```
Comment (TimLeach635) [inline src/main/resources/static/css/login.css]: Exactly one blank line at end of file
```suggestion
}
```

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Just some styling changes and one question!

File: src/main/resources/static/css/login.css (hunk 4 above)
Comment (Annie1790) [reply src/main/resources/static/css/login.css]: Hello @TimLeach635 ,

The reason was to remove the default border for input fields. Without it, it looks like this:
Screenshot from 2023-10-11 11-06-10

And with the transparent border:
Screenshot from 2023-10-11 11-01-50

Comment (TimLeach635) [reply src/main/resources/static/css/login.css]: Ah, I see! The preferred way of doing that would be `border: none;`

Comment (Annie1790) [reply src/main/resources/static/css/login.css]: I'll change it now then!

File: src/main/resources/static/css/login.css (hunk 5 above)
Comment (Annie1790) [reply src/main/resources/static/css/login.css]: I've added the blank line!

**Comment by Annie1790** [comment]
Hi @TimLeach635 , 
All good with the pull request now?

**Review by TimLeach635 — APPROVED** [review]

**Pull request merged**